ansible/testbed-cli.sh

```diff
@@ -240,6 +241,15 @@ function remove_topo
 {
   topology=$1
   passwd=$2
+  remove_keysight_api_server=0
+
+  if [ -n "$3" ]; then
+    echo $3
+    if [[ "remove_keysight_api_server" == "$3" ]]; then
+      remove_keysight_api_server=1
+    fi
```

Comment:
Can we use ptf_image_name field to decide action instead of extra parameter?

Reply:
ptf_image_name has been considered when keysight-api-server is to be removed. But as per the requirement by default remove_topo will not remove the keysight-api-server. Hence additional parameter is needed if to remove the api-server. So the ptf_image_name is used to identify it is a keysight-api-server and the extra parameter is to determine if to remove the api-server. An extra parameter is needed to provide the user of remove_topo a choice whether to remove the api-server of skip the removal.